tests/test_main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from random import randint
+
+
+from app.main import get_human_age
+
+
+def test_first_human_year():
+    cat_years = randint(0, 14)
```

Comment:
It's better not to use random in tests. Just choose concrete values from this range